util/publish-to-confluence.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+const { prompt } = require('inquirer');
+const path = require('path');
+const fs = require('fs');
+const fetch = require('node-fetch');
+const showdown = require('showdown');
+const error = require('../lib/error');
+
+const converter = new showdown.Converter(
+  {
+    parseImgDimensions: true,
+    simplifiedAutoLink: true,
+    tables: true
+  }
+);
+
+const CONFLUENCE_PAGES = './confluence-pages.json';
+
+const CONFLUENCE_DOMAIN = process.env.CONFLUENCE_DOMAIN;
+const CONFLUENCE_SPACE = process.env.CONFLUENCE_SPACE;
+const CONFLUENCE_USER = process.env.CONFLUENCE_USER;
+const CONFLUENCE_PASS = process.env.CONFLUENCE_PASS;
+
+async function gatherCreds () {
+  const answer = await prompt([
+    {
+      type: 'input',
+      name: 'domain',
+      message: 'Confluence domain:'
```

Comment:
Is this what they call the vanity/org name part of the host? I used `lifeomic.atlassian.com` because I thought "domain" referred to the complete domain name.

Reply:
yes, the vanity subdomain